src/connectors/radiotunes.ts

```diff
@@ -43,3 +45,27 @@ Connector.applyFilter(filter);
 function removeTrailingDash(text: string) {
 	return text.replace(/\s-\s$/, '');
 }
+
+Connector.getOriginUrl = () => {
+	let ta = Util.getTextFromSelectors(artistSelector);
+	const tt = Util.getTextFromSelectors(trackSelector);
+	const pta = Util.getTextFromSelectors('.now-playing-component__artist');
+	const ptt = Util.getTextFromSelectors('.now-playing-component__title');
```

Comment:
These variables could be renamed for improved readability - its not immediately celar what they are.